package.json

```diff
@@ -17,12 +17,15 @@
     "publish-gh-pages": "docusaurus deploy",
     "version": "docusaurus version",
     "rename-version": "docusaurus rename-version",
-    "update-api-reference": "docusaurus graphql-to-doc --force --pretty"
+    "update-api-reference": "docusaurus graphql-to-doc --force --pretty",
+    "prepare": "husky install"
```

Comment:
as far as i understand this should only be run once so we can remove it I guess, but where is husky dir with husky commands?

Reply:
@bmigirl you mean prepare script?